docs/src/main/docs/tracing/02_Zipkin.adoc

```diff
@@ -0,0 +1,86 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+= Zipkin Tracing
+:description: Helidon Tracing Support
+= :keywords: helidon, tracing
+
+== Zipkin Tracing
+Helidon SE is integrated with Zipkin tracer.
+
+The Zipkin builder is loaded through `ServiceLoader` and configured. You could
+also use the Zipkin builder directly, though this would create a source-code dependency
+on Zipkin tracer.
+
+
+=== Maven Coordinates
+To use Zipkin as the collector of traces (e.g. a Tracer),
+    add the following dependency to your project:
+
+[source,xml,subs="verbatim,attributes"]
+.Zipkin Tracer
+----
+<dependency>
+    <groupId>io.helidon.tracing</groupId>
+    <artifactId>helidon-tracing-zipkin</artifactId>
+</dependency>
+----
+
+=== Configuring Zipkin
+
+The following configuration options are supported by Zipkin tracer:
+
+|===
+|key            |default value      |builder method     |description
+
+|service name   |N/A                |serviceName        |name of the service, to distinguish traces crossing service boundaries
+|protocol       |http               |collectorProtocol  |protocol of the Zipkin trace collector
+|host           |localhost          |collectorHost      |host of the Zipkin trace collector
+|port           |9411               |collectorPort      |port of the Zipkin trace collector
+|path           |defined by version |collectorPath      |path of the Zipkin trace collector
+|api-version    |Version.V2         |version            |Zipkin specific method, set the prtocol version to communicate with trace collector
+|enabled        |true               |enabled            |if set to false, tracing would be disabled
+|tags           |N/A                |addTracerTag(String, String) |String tags to add to each span
```

Comment:
What does "each span" mean in the description of the `tags`, `boolean-tags` and `int-tags` keys ?

Reply:
"span" is a term used in tracing. It defines an event that has a start time and finish time, that can belong to a parent "span" and that can have 0 or more child "spans". There may be tags added to the span, and events logged. 
I do not describe the details, as if you use tracing, you understand the terms. 
This is documentation of how to use Zipkin tracer, not a general description of Open Tracing APIs.
If you feel this term glossary is needed, we should add it to the overall tracing documentation.

Reply:
And if your question is about the "each" part, then it really means each and every span reported to the trace collector.